testcafe/tests/drive_sharing.js

```diff
@@ -39,7 +34,7 @@ test('Drive : from Drive, go in a folder, upload a file, and share it', async t
   await drivePage.openActionMenu()
   await t.pressKey('esc') //close action Menu
 
-  await drivePage.uploadFiles([`../data/${file}`])
+  await drivePage.uploadFiles([`../data/${data.filePDF}`])
```

Comment:
We should keep the full path into the variable, WDYT?

Reply:
I added 2 vars : one for file name, and one for the data folder path, as I re-used the file name later (and data folder path is always the same anyway!)